flink-core/src/main/java/org/apache/flink/core/FlinkVersion.java

```diff
@@ -16,7 +16,7 @@
  * limitations under the License.
  */
 
-package org.apache.flink.testutils.migration;
+package org.apache.flink.core;
```

Comment:
For FLIP-197 this enum must be accessible for `@PublicEvolving` which is in the `flink-annotation` module. Unless we let `flink-annotation` depend on `flink-core`. @tillrohrmann what is your preference here?

We could move the class to the module but does the package `org.apache.flink.annotation` fit? Alternatively, we could also move this to a package `org.apache.flink.FlinkVersion` in the `flink-annotation` module.

Reply:
But I can also move it over once I get to implementing FLIP-197.

Reply:
@tillrohrmann No problem, please, I've already moved to `flink-annotations`.